src/Main.java

```diff
@@ -9,7 +9,7 @@ public static void main(String[] args) {
         try {
             firstShoppersCart.addProduct( sneakers );
         } catch (Exception e) {
-            System.out.println(e.getMessage());
+            System.out.println("send me location");
```

Comment:
Looks good but let's change it slightly to align with our use-case.

```suggestion
            System.out.println("An exception happened while adding a product to the cart.");
```

Reply:
Alright looks good. Thanks